packages/paste-website/src/pages/foundations/data-visualization/index.mdx

```diff
@@ -66,7 +65,20 @@ This foundation page was created to help establish a familiar and accessible use
 
 The easiest way to use the data visualization tokens in your charts is to use `usePasteHighchartsTheme` from the [data visualization library package](/core/libraries/data-visualization) with [Highcharts](https://www.highcharts.com/). The `usePasteHighchartsTheme` hook takes an object of Highchart configurations and returns a new object with Paste colors and fonts.
 
-<LivePreview scope={{Highcharts, HighchartsReact, usePasteHighchartsTheme, LineChartOptions}} language="jsx" noInline>
+We strongly advise that you include the [Highcharts Accessibility module](/foundations/data-visualization#adding-highcharts-accessibility-module) to ensure that your charts are accessible to all UserMessage.
```

Comment:
question: What's UserMessage?

Reply:
Sorry that's my copilot going crazy chaging user to a variable. fixed in upcoming commit